From: Ray Speth <speth@mit.edu>
Date: Wed, 16 Nov 2022 22:51:46 -0500
Subject: [Ubuntu] Drop unsupported pip flag

---
 interfaces/cython/SConscript | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/interfaces/cython/SConscript b/interfaces/cython/SConscript
index 07d383a..8909146 100644
--- a/interfaces/cython/SConscript
+++ b/interfaces/cython/SConscript
@@ -107,7 +107,7 @@ ext = localenv.LoadableModule(f"cantera/_cantera{module_ext}",
                               obj, LIBPREFIX="", SHLIBSUFFIX=module_ext,
                               SHLIBPREFIX="", LIBSUFFIXES=[module_ext])
 
-build_cmd = ("$python_cmd_esc -m pip wheel -v --no-build-isolation --use-feature=in-tree-build --no-deps "
+build_cmd = ("$python_cmd_esc -m pip wheel -v --no-build-isolation --no-deps "
              "--wheel-dir=build/python/dist build/python")
 plat = info['plat'].replace('-', '_').replace('.', '_')
 wheel_name = (f"Cantera-{env['cantera_version']}-cp{py_version_nodot}"
@@ -172,7 +172,7 @@ if env["stage_dir"]:
 
     install_cmd.append(f"--root={stage_dir.resolve()}")
 
-install_cmd.extend(("--no-build-isolation", "--use-feature=in-tree-build", "--no-deps", "-v", "--force-reinstall",
+install_cmd.extend(("--no-build-isolation", "--no-deps", "-v", "--force-reinstall",
                     "build/python"))
 if localenv['PYTHON_INSTALLER'] == 'direct':
     mod_inst = install(localenv.Command, 'dummy', mod,
